src/main/java/org/spongepowered/common/mixin/api/mcp/entity/passive/VillagerTrades_EmeraldForItemsTradeMixin_API.java

```diff
@@ -44,23 +43,25 @@
 // added as the only thing needing to be done is a simple default implementation
 // with an empty MerchantRecipeList and diff the list with an empty one and
 // provide the resulting diff'ed MerchantRecipe (TradeOffer) as the result.
-@Mixin(VillagerEntity.EmeraldForItems.class)
-public class EntityVillager_EmeraldForItemsMixin_API implements TradeOfferGenerator {
+//
+// i509VCB: Yes the classes which implement ITrade, like ItemsForEmeraldsAndItemsTrade are package private
+// For clarification this trade is multiple Items -> 1 Emerald
+// TODO: These need a new home
+@Mixin(targets = "net/minecraft/entity/merchant/villager/VillagerTrades$EmeraldForItemsTrade")
```

Comment:
As per the target, this would go in...

api.mcp.entity.merchant.villager

Reply:
Yeah I'll move it tomorrow afternoon. Hopefully a conflict doesn't break anything in the next 16 hours.